### ISSUE TITLE
which function I can use to extract the pixel data?

### ISSUE THREAD

**Issue by trannhuphuan** [cornerstonejs/cornerstoneWADOImageLoader#205]
Dear cornerstoneWADOImageLoader,

I want to use the dicom image which user upload. It means that I want to download the image after anonymized from client to server. Which function of cornerstoneWADOImageLoader.js can I use for this?
I try to request the image object to server, but it is difficult to convert this object image to numpy array. So, which function I can use for this problem?

Thank you so much.

**Comment by GleasonBian** [comment]
under link maybe have you require messages
https://github.com/GleasonBian/DicomViewer
https://github.com/GleasonBian/CornerstoneVueWADO

**Comment by dannyrb** [comment]
@trannhuphuan, I'm sorry this response is coming so late. If you still require assistance, please feel free to create a new issue. I believe the delay is partly because your question is a bit hard to follow. Any information around what you've tried so far, or a minimal example, will make it much easier for us to respond in a timely manner.
